### DISCUSSION TITLE
Details on 'championnat'

### DISCUSSION THREAD

**Discussion by clapoujade** [eric-taix/volley34-mobile#229] [Ideas]
I have the feeling that the button 'afficher les détails' is not useful.
The current Ux is a bit confusing to me as the button only appears when layout is vertical.... and the user must click somewhere else (Android back button) to get back to light view...which also change the layout back to vertical....confusing.

To me, the light and extented view should be tied to the layout direction, by having an automatic switch (between light and extented view) when the user turns the mobile. 
What about having just a kind of tip ' 🔄 pour détails' when layout is vertical

**Comment by eric-taix** [comment]
I totally agree with you. Could you create an issue from this discussion?
